### ISSUE TITLE
Scenario description not displayed during test run

### ISSUE THREAD

**Issue by IntergalacticPenguin** [SmartBear/cucumberstudio-issue-tracker#224]
### Steps to reproduce
1. Create scenario with description and no steps
2. Create and populate test run with that scenario
3. Run test execution cycle
-> Observe scenario description not being displayed in some cases.

### Expected behaviour
Scenario description should appear at all times during test run execution.

### Actual behaviour
Sometimes scenario description is not displayed in test run popup window.
It is visible in scenario detail in test run and in scenario edit form.

### My configuration
**Browser: Version 83.0.4103.116 (Official Build) (64-bit)**

**Operating system: macOS Catalina 10.15.4 (19E287)**

![Screenshot 2020-07-03 at 12 59 52](https://user-images.githubusercontent.com/18715547/86465934-ef5bb280-bd32-11ea-9e08-f1f4fe647b4f.png)
![Screenshot 2020-07-03 at 13 00 06](https://user-images.githubusercontent.com/18715547/86465937-f1257600-bd32-11ea-83d0-73cf7d6d401b.png)

**Comment by HoudaDabou** [comment]
Hi!

I need to check your project so could you please grant access to CucumberStudio support so I can access your project and investigate your issue?

This option is available from the help menu at the bottom left of the screen (this option is only available for project admin):

![grant_access_userCS](https://user-images.githubusercontent.com/43133237/86778944-462afa00-c05b-11ea-8352-594ecd7c9cb3.png)

**Comment by IntergalacticPenguin** [comment]
Hello, thank you for your response!

I have granted you access for the project ID: 169494

**Comment by HoudaDabou** [comment]
Hi again,

Thanks :+1: 

So I checked your project and it seems that all your existing scenarios don't contain any definition. 

Also, I checked the logs and noticed some performed actions on your scenarios like updating the names or the description, and cloning or moving the scenarios.

I need more details about who performed these actions and about your workflow. Since this information is sensitive data, could you please email us at: studio@cucumber.io in order to continue investigating your issue?


Thank you!

**Comment by aurelien-reeves** [comment]
Hello @IntergalacticPenguin 

I am investigating your issue.

First, did you synchronize your test-runs after adding descriptions to your scenarios?

**Comment by IntergalacticPenguin** [comment]
Hello, definitely had synchronized test run scenarios.
I was able to see the description in test run scenario detail, but it wasn't shown in Test Run popup window.

**Comment by aurelien-reeves** [comment]
Ok.

I continue investigation. 

If it happen again, please contact the support using the in-app chat as we can be notified more quickly and work live with you on the issue, that would be easier to investigate.

**Comment by IntergalacticPenguin** [comment]
I have encountered this issue again. 
I just created test run and ran a few tests, the descriptions were visible at first, but then stopped appearing.

I tried to contact you using chat, but I didnt find the contact button.

![Screenshot 2020-08-12 at 09 52 02](https://user-images.githubusercontent.com/18715547/89989891-daebcd80-dc81-11ea-8f5f-9784731e9e58.png)
![Screenshot 2020-08-12 at 09 52 18](https://user-images.githubusercontent.com/18715547/89989897-dcb59100-dc81-11ea-9c18-094f0783fa88.png)
